solidity/test/BondedECDSAKeepFactoryTest.js

```diff
@@ -1491,6 +1491,27 @@ describe("BondedECDSAKeepFactory", function () {
     })
   })
 
+  describe("setMinimumBondableValue", async () => {
+    before(async () => {
+      await initializeNewFactory()
+      await initializeMemberCandidates()
+    })
+
+    it("fails for unknown application", async () => {
```

Comment:
Can we add such a case for `registerMemberCandidate`? I believe we missed it before.

Reply:
Sure! Added in https://github.com/keep-network/keep-ecdsa/pull/508/commits/74a261f8753517f629501f6c8c2d749c832edcb5.